geo/src/algorithm/line_measures/metric_spaces/geodesic.rs

```diff
@@ -1,16 +1,120 @@
 use super::super::{Bearing, Destination, Distance, InterpolatePoint};
 use crate::Point;
 use geographiclib_rs::{DirectGeodesic, InverseGeodesic};
+use std::sync::LazyLock;
 
-/// An ellipsoidal model of the earth, using methods given by [Karney (2013)].
+/// Use the [`Geodesic`] constant (an instance of `GeodesiceMeasure`) rather than building your own
+/// customized [`GeodesicMeasure`] for standard ellipsoidal Earth measurements.
 ///
-/// Distances are computed using [geodesic lines] and are measured in meters.
+/// Measures distance on an ellipsoidal model of the Earth using methods given by [Karney (2013)].
+///
+/// Geodesic measurements are more accurate than [`Haversine`], but a bit slower, because they
+/// account for the fact that the Earth is not a perfect sphere.
+///
+/// Distances are computed using [geodesic lines] and are measured **in meters**.
+///
+/// [`GeodesicMeasure`] allows creating a geodesic model with custom values for the equatorial
+/// radius (`A`) and the inverse flattening factor (`F`). Unless you have special needs, just use
```

Comment:
The most minor of nits: we should try not to use "just" in our public docs IMO, as it assumes a degree of knowledge / familiarity that the reader may not have (we are actually **very** good about this, barring some instances floating around in `Haversine` examples)

Reply:
This is a good nit. "just" is something I'm largely trying to strike from my writing. It usually doesn't add anything, is sometimes ambiguous, and sometimes condescending.

I've removed a few other instances in https://github.com/georust/geo/pull/1311/commits/1d9005e075f33b8c946859bac80423c334fe5424